### ISSUE TITLE
Shebang should use python3

### ISSUE THREAD

**Issue by julianbrost** [kbytesys/pynagsystemd#2]
The shebang should be `#!/usr/bin/env python3` as the script is written for Python 3 and most distributions ship Python 2 as `python` command. [PEP 394](https://www.python.org/dev/peps/pep-0394/) states that `python3` should always exists for Python 3.

**Comment by kbytesys** [comment]
The script working fine also on python2. 

The readme is outdated, but the setup.py is correct and pip sets the proper shebang after the setup phase.

Have you had any particular issue because of the shebang?

**Comment by julianbrost** [comment]
I don't think it works fine on Python 2 as there is no `FileNotFoundError`:

```
$ PATH= ./bin/pynagsystemd.py
Traceback (most recent call last):
  File "./bin/pynagsystemd.py", line 4, in <module>
    __import__('pkg_resources').run_script('pynagsystemd==1.0.2', 'pynagsystemd.py')
  File "/tmp/py2/lib/python2.7/site-packages/pkg_resources/__init__.py", line 719, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/tmp/py2/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1511, in run_script
    exec(script_code, namespace, namespace)
  File "/tmp/py2/lib/python2.7/site-packages/pynagsystemd-1.0.2-py2.7.egg/EGG-INFO/scripts/pynagsystemd.py", line 63, in <module>

  File "/tmp/py2/lib/python2.7/site-packages/pynagsystemd-1.0.2-py2.7.egg/EGG-INFO/scripts/pynagsystemd.py", line 59, in main

  File "/tmp/py2/lib/python2.7/site-packages/nagiosplugin-1.2.4-py2.7.egg/nagiosplugin/check.py", line 120, in main
    runtime.execute(self, verbose, timeout)
  File "/tmp/py2/lib/python2.7/site-packages/nagiosplugin-1.2.4-py2.7.egg/nagiosplugin/runtime.py", line 130, in execute
    self.run(check)
  File "/tmp/py2/lib/python2.7/site-packages/nagiosplugin-1.2.4-py2.7.egg/nagiosplugin/runtime.py", line 117, in run
    check()
  File "/tmp/py2/lib/python2.7/site-packages/nagiosplugin-1.2.4-py2.7.egg/nagiosplugin/check.py", line 105, in __call__
    self._evaluate_resource(resource)
  File "/tmp/py2/lib/python2.7/site-packages/nagiosplugin-1.2.4-py2.7.egg/nagiosplugin/check.py", line 72, in _evaluate_resource
    metrics = resource.probe()
  File "/tmp/py2/lib/python2.7/site-packages/pynagsystemd-1.0.2-py2.7.egg/EGG-INFO/scripts/pynagsystemd.py", line 26, in probe

NameError: global name 'FileNotFoundError' is not defined
```

**Comment by kbytesys** [comment]
Ops! I will fix that in few days because I'm using this script in some servers without python3.

**Comment by kbytesys** [comment]
Fixed in the 1.0.3 release. Thanks!
